blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -118,7 +124,13 @@ export class MultiBranch extends Component {
                         })
                         }
                     </Table>
+                    {branches.$pager &&
+                        <button disabled={!branches.$pager.hasMore} className="btn-show-more btn-secondary" onClick={() => branches.$pager.fetchMore()}>
```

Comment:
@tfennelly no, do you have a better idea? I'm happy to replace them with underscores or something else, this is to not conflict with other properties.